dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3790,6 +3801,279 @@ def test_null_row_stats_correct_after_updates(self, *mocks):
         # recognized as object, not float64
         self.assertSetEqual({8}, profile._profile[1].null_types_index["None"])
 
+        # Tests row stats disabled
+        options = StructuredOptions()
+        options.row_statistics.is_enabled = False
+        options.multiprocess.is_enabled = False
+        profile2 = StructuredProfiler(data1, options=options)
+        self.assertEqual(0, profile2.row_is_null_count)
+        self.assertEqual(0, profile2.row_has_null_count)
```

Comment:
For duplicate etc, we use `None` should we expect the same here?

Reply:
these will still be zero, they are attributes and not the method call so they will default to 0. One thought, should we change the init of `row_is_null_count` and `row_has_null_count` to be `None` vs 0?

Reply:
Yes, I think we did that with unique as well.

Reply:
I believe this still needs to be updated like we did for unique_rows and duplicate rows where it is none when disabled.